### PR TITLE
Random shabad

### PR DESCRIPTION
**Pull request by inderpreetsingh**
* Adds Random Shabad
* Adds Hukamnama from Sri Darbar Sahib 
* Fixes ShabadUndefined Problem.
* Goes hand in hand with https://github.com/KhalisFoundation/sttm-desktop/pull/99
randomshabad

### CODE REVIEW

File: js/menu.js [hunk 1]
```diff
@@ -18,13 +19,55 @@ const customSlidesButton = h(
...
+      } },
+    h('i.fa.fa-random.list-icon'),
+    'Show Random Shabad'));
+const hukamnamaButton = h(
+  'li',
+  h(
+    'a.hukamnama-button',
+    {
+      onclick: () => {
+        getJSON('https://api.gurbaninow.com/v2/hukamnama/today', (error, response) => {
```
Comment (navdeepsinghkhalsa) [inline js/menu.js]: Could you please use https://api.banidb.com/hukamnama/today? SSL will be sorted out by tomorrow

File: js/menu.js [hunk 2]
```diff
@@ -18,13 +19,55 @@ const customSlidesButton = h(
...
       module.exports.toggleMenu('#custom-slides-page');
     } },
   h('i.fa.fa-times'));
+const randomShabadButton = h(
+  'li',
+  h(
+    'a.random-shabad-button',
+    {
+      onclick: () => {
+        global.core.search.loadShabad(Math.round(Math.random() * 5540));
```
Comment (navdeepsinghkhalsa) [inline js/menu.js]: I will update the in-app search API to do random search

File: js/menu.js (hunk 1 above)
Comment (inderpreetsingh) [reply js/menu.js]: Okay, sure! Doing that.

File: js/menu.js (hunk 2 above)
Comment (inderpreetsingh) [reply js/menu.js]: Do you mean via sql in `search.js`?  
Pardon me if that's a noob question 🤓

Comment (navdeepsinghkhalsa) [reply js/menu.js]: Correct

**Comment by inderpreetsingh** [comment]
@navdeepsinghkhalsa  veerji, pushed commits to get random shabad from API. Let me know if there's anything missing. 

Cleanup of some of menu.js is in other PR.

**Pull request merged**